prow/tide/tide.go

```diff
@@ -1410,15 +1403,14 @@ func (c *Controller) dividePool(pool map[string]PullRequest, pjs []prowapi.ProwJ
 		}
 		sps[fn].prs = append(sps[fn].prs, pr)
 	}
-	for _, pj := range pjs {
-		if pj.Spec.Type != prowapi.PresubmitJob && pj.Spec.Type != prowapi.BatchJob {
-			continue
-		}
-		fn := poolKey(pj.Spec.Refs.Org, pj.Spec.Refs.Repo, pj.Spec.Refs.BaseRef)
-		if sps[fn] == nil || pj.Spec.Refs.BaseSHA != sps[fn].sha {
-			continue
+
+	for subpoolkey, sp := range sps {
+		pjs := &prowapi.ProwJobList{}
+		if err := c.prowJobClient.List(c.ctx, pjs, ctrlruntimeclient.MatchingField(BaseSHAIndexKey, sp.sha)); err != nil {
```

Comment:
Is it ok that we no longer specify `ctrlruntimeclient.InNamespace(c.config().ProwJobNamespace)`? Do we need to somehow compose the MatchingField and InNamespace requirements?

Reply:
Added it again